apps/bare-expo/android/settings.gradle

```diff
@@ -7,6 +7,8 @@ applyNativeModulesSettingsGradle(settings)
 
 include(":expo-image")
 project(":expo-image").projectDir = new File("../../../packages/expo-image/android")
+include(":unimodules-test-core")
+project(":unimodules-test-core").projectDir = new File("../../../packages/unimodules-test-core/android")
```

Comment:
Would it be required for builds and not tests? In other words, will users need to add this in their projects?

Reply:
No. This is added to bare-expo only because it does not used published version of our modules, but the one from repository.